rts/motoko-rts/src/gc/incremental/sort.rs

```diff
@@ -8,34 +8,40 @@ pub fn sort<Compare: Fn(usize, usize) -> Ordering>(array: &mut [usize], compare:
     }
 }
 
+/// Special version of quicksort by Niklaus Wirth: https://people.inf.ethz.ch/wirth/AD.pdf, pages 65/66.
```

Comment:
Thanks for commenting and renaming this!

Reply:
Thank you very much for reviewing. And also thanks for suggesting to improve the naming.